processing/src/main/java/org/apache/druid/query/groupby/GroupByQuery.java

```diff
@@ -565,7 +565,7 @@ private boolean canDoLimitPushDown(
    * limit/order spec (unlike non-push down case where the results always use the default natural ascending order),
    * so when merging these partial result streams, the merge needs to use the same ordering to get correct results.
    */
-  private Ordering<ResultRow> getRowOrderingForPushDown(
+  private OrderingAndDimensions getRowOrderingAndDimensionsForPushDown(
```

Comment:
note: I think you could keep the old method with its signature; and service it like: `return getOrderingAndDimensions(false).getRowOrdering()`